core/src/main/scala/kafka/log/LogManager.scala

```diff
@@ -802,7 +814,7 @@ class LogManager(logDirs: Seq[File],
         // Now that replica in source log directory has been successfully renamed for deletion.
         // Close the log, update checkpoint files, and enqueue this log to be deleted.
         sourceLog.close()
-        checkpointLogRecoveryOffsetsInDir(sourceLog.dir.getParentFile)
+        checkpointLogRecoveryOffsetsInDir(sourceLog.dir.getParentFile, Some(Seq(sourceLog)))
```

Comment:
Since sourceLog will be deleted soon, there is no need to clean the snapshot for this partition. We could just pass in None.

Reply:
I think it'd better pass it explicitly since only passing None could also means doing for all logs.